W-15612083-Run-VALE-on-latest-Studio-repo-CP (#524)
mulesoft/docs-studio: modules/ROOT/pages/api-search.adoc

include::_attributes.adoc[]
endif::[]

When you import your API specifications from Exchange or Maven as dependencies of your projects, those dependencies relocate from `src/resources/api/` to a ZIP file in your project's classpath. This change means that you can't search for your API dependencies by using the File Search option in Studio. To enable you to search for specific text inside your API dependencies, Studio introduces the API Search feature.

> You can access the API Search menu from the taskbar at the Anypoint Studio display, either by selecting *Search* > *Search...* or by using the control and H keys on your keyboard simultaneously:

Reviewer: ```suggestion
You can access the API Search menu from the taskbar at the Anypoint Studio display, either by selecting *Search* > *Search...* or by pressing `Ctrl+h`:
```

Author: OK! Thanks!

* *Local API Files* searches inside API specifications added from a local file or from Anypoint Design Center.
. Click *Search*.

> You can also search for all API specifications inside your open projects. You can select *Navigate* > *Open API Resource* in the taskbar at the Anypoint Studio display, or use `cmd+Shift+P` (MacOS) or `control+shift+P` (Windows/Linux) keyboard shortcuts:

Reviewer: ```suggestion
You can also search for all API specifications inside your open projects. You can select *Navigate* > *Open API Resource* in the taskbar at the Anypoint Studio display, or press `Cmd+Shift+p` (MacOS) or `Ctrl+Shift+P` (Windows/Linux):
```
